src/agent/filecontainerhelper.ts

```diff
@@ -13,6 +14,29 @@ export function copyToFileContainer(taskCtx: ctxm.TaskContext, localPath: string
 	return fc.copyToFileContainer(localPath, containerId, containerFolder);
 }
 
+export function getUploadHeaders(isGzipped: boolean, 
```

Comment:
x-vso-contentId is a special one that requires custom logic on the client side.  It's why there's a FileContainerHelper in the first place.  I'm not sure it's worth trying to get the generator to do anything with it.